migrate to readthedocs documentation (#311)
jupyter-server/jupyter-scheduler: docs/users/index.md

# Users

> These pages are targeted at people interested in simply installing Jupyter

Reviewer: ```suggestion
These docs are intended for users interested in installing Jupyter
```

> These pages are targeted at people interested in simply installing Jupyter
> Scheduler and using it.

Reviewer 2: ```suggestion
These pages are for people interested in installing and using Jupyter Scheduler.
```

> For configuration options, please refer to our {doc}`operator's guide </operators/index>`.

Reviewer 2: Link to developer's guide?

## Installation

> Jupyter Scheduler can be installed from the PyPi registry via `pip`:

Reviewer: ```suggestion
Jupyter Scheduler can be installed from the PyPI registry via `pip`:
```

```
pip install jupyter-scheduler
```

This automatically enables its extensions. You can verify this by running

```
jupyter server extension list
jupyter labextension list
```

> and asserting that both the `jupyter_scheduler` server extension and the

Reviewer 2: ```suggestion
and checking that both the `jupyter_scheduler` server extension and the
```
